modules/mapping-utils/src/mappingToDisplayTreeData.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+const MOCK_MAPPING = {
+  boolean: {
+    type: 'boolean',
+  },
+  children: {
+    properties: {
+      key: {
+        type: 'text',
+        fields: {
+          keyword: {
+            type: 'keyword',
+            ignore_above: 256,
+          },
+        },
+      },
+    },
+  },
+  float: {
+    type: 'float',
+  },
+  int: {
+    type: 'long',
+  },
+  stringarray: {
+    type: 'text',
+    fields: {
+      keyword: {
+        type: 'keyword',
+        ignore_above: 256,
+      },
+    },
+  },
+  text: {
+    type: 'text',
+    fields: {
+      keyword: {
+        type: 'keyword',
+        ignore_above: 256,
+      },
+    },
+  },
+};
+const elasticMappingToDisplayTreeData = elasticMapping => {
```

Comment:
This function exists twice,  can you use the same one in both spots?

Reply:
I added it here so it can be shared through the `mapping-utils` module, but I think we need to publish a new version of the module to have access to it in `components.